src/Microsoft.SqlTools.ServiceLayer/SqlAssessment/Contracts/AssessmentRequest.cs

```diff
@@ -1,129 +1,129 @@
-        public string RulesetVersion { get; set; }
-
-        /// <summary>
-        /// Gets or sets assessment ruleset name
-        /// </summary>
-        public string RulesetName { get; set; }
-
-        /// <summary>
-        /// Gets or sets assessed target's type.
-        /// Supported values: 1 - server, 2 - database.
-        /// </summary>
-        public SqlObjectType TargetType { get; set; }
-
-        /// <summary>
-        /// Gets or sets the assessed object's name. 
-        /// </summary>
-        public string TargetName { get; set; }
-
-        /// <summary>
-        /// Gets or sets check's ID.
-        /// </summary>
-        public string CheckId { get; set; }
-
-        /// <summary>
-        /// Gets or sets tags assigned to this item.
-        /// </summary>
-        public string[] Tags { get; set; }
-
-        /// <summary>
-        /// Gets or sets a display name for this item.
-        /// </summary>
-        public string DisplayName { get; set; }
-
-        /// <summary>
-        /// Gets or sets a brief description of the item's purpose.
-        /// </summary>
-        public string Description { get; set; }
-
-        /// <summary>
-        /// Gets or sets a <see cref="string"/> containing
-        /// an link to a page providing detailed explanation
-        /// of the best practice.
-        /// </summary>
-        public string HelpLink { get; set; }
-
-        /// <summary>
-        /// Gets or sets a <see cref="string"/> indicating
-        /// severity level assigned to this items.
-        /// Values are: "Information", "Warning", "Critical".
-        /// </summary>
-        public string Level { get; set; }
-    }
-
-    /// <summary>
-    /// Generic SQL Assessment Result
-    /// </summary>
-    /// <typeparam name="T">
-    /// Result item's type derived from <see cref="AssessmentItemInfo"/>
-    /// </typeparam>
-    public class AssessmentResultData<T>
-        where T : AssessmentItemInfo
-    {
-        /// <summary>
-        /// Gets the collection of assessment results.
-        /// </summary>
-        public List<T> Items { get; } = new List<T>();
-
-        /// <summary>
-        /// Gets or sets SQL Assessment API version.
-        /// </summary>
-        public string ApiVersion { get; set; }
-    }
-
-    /// <summary>
-    /// Generic SQL Assessment Result
-    /// </summary>
-    /// <typeparam name="T">
-    /// Result item's type derived from <see cref="AssessmentItemInfo"/>
-    /// </typeparam>
-    public class AssessmentResult<T> : AssessmentResultData<T>
-        where T : AssessmentItemInfo
-    {
-        /// <summary>
-        /// Gets or sets a value indicating
-        /// if assessment operation was successful.
-        /// </summary>
-        public bool Success { get; set; }
-
-        /// <summary>
-        /// Gets or sets an status message for the operation.
-        /// </summary>
-        public string ErrorMessage { get; set; }
-    }
-}
+//
```

Comment:
Please undo all these other changes so we can easily see what's actually changed